docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+# Intended for local development, use at your own risk
```

Comment:
já bych sem asi docker compose nedával, at si to každej rozjede jak chce na svým dockeru / bez dockeru

když to tu necháš, měl bys nějak detailně popsat jak s tím pracvat

když pustím compoer install uvnitř dockeru, tak to nic nedělá 

![image](https://github.com/3BRS/sylius-payment-restrictions-plugin/assets/6840281/3e620f80-26b0-4b62-8c7d-64546e6776fa)

Reply:
A co to dát jako `docker-compose.dev.yml` nebo něco podobně neškodného?

Nám se to osvědčilo, protože přijde nováček a já mu řeknu `docker compose up` a může pracovat.

> když pustím compoer install uvnitř dockeru, tak to nic nedělá

Je tam aktivní XDebug, nechytil se ti XDebug v IDE? Třeba v jiném okně, to si pak ani nevšimnu, kde se XDebug zastavil.

Teď jsem to zkoušel (a i jsem to v tom předtím testoval) a `composer install` proběhl.

Reply:
`docker-compose.yml` přejmenován na `docker-compose.example.yml`